structure/0072-legal-team.md

```diff
@@ -0,0 +1,96 @@
+# RFC 4269: Legal Team
+
+This document has been written according to the process outlined in [RFC 0006](https://github.com/QuiltMC/rfcs/blob/main/structure/0006-governance.md#teams).
+
+# Summary
+
+The Legal Team is a standalone team in charge of handling legal matters to the best of its ability. This team will not require its members to be lawyers 
+or have any form of training, although this would be preferred. 
+
+It is made clear that this team will not be providing legal guidance but only advices to the best of their ability and team members cannot be held
+liable for their work in good faith as a Legal Team member.
+
+# Motivation
+
+Historically, legal matters have been handled by various different people without any proper structure. The de-facto team who would handle particular 
+matters (eg. the Admin Board for registering Quilt as a non-profit) may feel unqualified to handle those issues and/or may not have the bandwidth for it. 
+Moreover, someone handling legal matters leaving that de-facto team should not prevent them from handling those matters, especially 
+considering the current scarcity of trained or knowledgeable members.
+
+Some laws may also require us to have a designated legal team, such as to designate a Data Protection Officer 
+([Art. 37 EU GDPR](https://gdpr-info.eu/art-37-gdpr/)).
+
+# Explanation
+
+## Responsibilities
+
+The team is responsible for the following:
+
+- Ensuring Quilt stays up to date with data regulation laws (eg. EU GDPR, COPPA)
+    - Keeping the privacy policies and terms of services up to date
+    - Designating a DPO
+    - Responding to data privacy requests
+    - Assisting the Infrastructure team in staying compliant
+- Reviewing contracts signed by the organisation
+- Handling registration of the organisation as a legal entity, if decided
+- Advising other teams on any legal issues that may arise
+- Answering questions and concerns from community members to the best of their ability
+- Writing guidelines on what changes to Quilt projects require legal verification and providing a process for this verification to happen
+- Writing internal or public processes on how to handle requests
+- Any other legal matter not explicitly listed in this document
+
+## Legal Team Server
+
+The Legal Team has a private server not viewable by any other team. This server must only be used for discussing problems 
+which need the uttermost privacy, as required by law. This includes, for example, talking about an ongoing data request or DMCA takedown
+without any name being redacted.
+
+While this deviates from the usual observable approach that makes sure every team can be monitored by at least one other independent team, 
+privacy is legally required to handle some matters, such as data privacy requests.
+
+The server shall be owned by one of the Legal Team leads, who is in charge of managing the server. 
```

Comment:
Given that this precludes oversight of the Legal Team's decisions or actions by other teams/organization members, should there be a required (or at least highly recommended) minimum of two members in the Legal Team, such that there is some level of accountability with respect to each other?

Reply:
I'm thinking this from the angle of accountability and mutual oversight: having/recommending/requiring another person on the Legal Team means that:
 - If the first person is absent or otherwise preoccupied, there is another who can act as well;
 - Information is known to both (as much as applicable laws allow) rather than sequestered only to a single person;
 - Each member can [over]see the actions of the other, and give their recommendations/opinions whilst retaining privacy of the matter to within the Legal Team.

It might be too much to some, but I think there is some value to requiring mutual oversight within the team, since the Legal Team is otherwise unchecked in discussions/decisions by other teams (outside of the required transparency reports and Administrative Board arbitration).

Reply:
I can add it as a recommendation if you really want, but that cannot be a strict rule. The legal team still has to function even if there is a single member in it.

Reply:
Fair enough then.